src/components/ResultButton/ResultButton.js

```diff
@@ -1,8 +1,8 @@
 import styled from 'styled-components/macro'
 
-const ResultButton = ({ toggleResultView }) => {
+const ResultButton = ({ openResultView }) => {
   const handleResultView = () => {
-    toggleResultView()
+    openResultView()
   }
   return (
     <StyledResultButton onClick={handleResultView}>
```

Comment:
Why do you add an extra function `handleResultView` when you can actually call it directly?
```suggestion
    <StyledResultButton onClick={openResultView}>
```